src/server/src/main/java/io/cassandrareaper/resources/ClusterResource.java

```diff
@@ -184,10 +184,26 @@ public Response addCluster(
     }
     Optional<Cluster> existingCluster = context.storage.getCluster(newCluster.getName());
     if (existingCluster.isPresent()) {
-      LOG.info("cluster already stored with this name: {}", existingCluster);
-      return Response.status(403)
-          .entity(String.format("cluster \"%s\" already exists", existingCluster.get().getName()))
-          .build();
+      LOG.info(
+          "Cluster already stored with this name: {}. Trying to updating the node list.",
+          existingCluster.get().getName());
+      Cluster updatedCluster = updateClusterSeeds(existingCluster.get(), seedHost.get());
+      if (updatedCluster.getSeedHosts().equals(existingCluster.get().getSeedHosts())) {
+        LOG.info("Node list of cluster {} is already up to date.", existingCluster.get().getName());
+        return Response.notModified()
```

Comment:
`NOT_MODIFIED` isn't mean to be used in this situation. It is only for responding to conditional-GETS.

This has been addressed in #401